coalib/processes/Processing.py

```diff
@@ -380,6 +397,12 @@ def instantiate_processes(section,
                              and the arguments passed to each process which are
                              the same for each object.
     """
+    try:
+        debug_bears = False if section['debug_bears'].value=='False' else (
+            section['debug_bears'].value)
+    except IndexError:
+        debug_bears = False
```

Comment:
maybe do an `'debug_bears' in section` inside the above `if` expression? Would compactify code a bit more. Don't worry: Due to Python's short circuitance following conditionals won't be executed if `'debug_bears' in section` evaluates to `False`

Reply:
You are msising the `'debug_bears' in section` check, otherwise there might be lookup errors if the key doesn't exists:
```python
debug_bears = (False
               if 'debug_bears' in section and section['debug_bears'].value == 'False' else
               typed_list(str)(section['debug_bears'].value))
```